ironfish-rust/src/transaction/mod.rs

```diff
@@ -707,25 +712,25 @@ fn value_balance_to_point(value: i64) -> Result<ExtendedPoint, IronfishError> {
     Ok(value_balance.into())
 }
 
-/// Confirm that balance of input and output values is consistent with
-/// those used in the proofs.
+/// Calculate balance of input and output values.
 ///
 /// Does not confirm that the transactions add up to zero. The calculation
 /// for fees and change happens elsewhere.
-fn check_value_consistency(
-    public_key: &PublicKey,
+fn calculate_value_balance(
     binding_verification_key: &ExtendedPoint,
     value: i64,
-) -> Result<(), IronfishError> {
+    burns: &[BurnDescription],
+) -> Result<ExtendedPoint, IronfishError> {
     let value_balance_point = value_balance_to_point(value)?;
 
-    let calculated_public_key = binding_verification_key - value_balance_point;
+    let mut public_key = binding_verification_key - value_balance_point;
```

Comment:
Why is the balance called a `public_key`?

Reply:
Ah, that probably doesn't make sense anymore, or never really did I guess. Eventually it _is_ used as a public key (https://github.com/iron-fish/ironfish/pull/2769/files#diff-27fd1eb5ae888f12e53e1f310371941c240868ada1d0af6f33004d1f32acb74eR683), but naming it such here is not ideal. Will update it to something better